projects/arlas-toolkit/src/lib/services/export-csv/export-csv.service.ts

```diff
@@ -28,26 +28,32 @@ import { Observable } from 'rxjs';
 import { map, mergeMap } from 'rxjs/operators';
 import { ArlasSettingsService } from '../settings/arlas.settings.service';
 import { ArlasCollaborativesearchService, ArlasConfigService } from '../startup/startup.service';
+import { ArlasCollectionService } from '../collection/arlas-collection.service';
 
 @Injectable({
   providedIn: 'root'
 })
 export class ArlasExportCsvService {
 
   public constructor(private collaborativesearchService: ArlasCollaborativesearchService, private configService: ArlasConfigService,
-    private settingsService: ArlasSettingsService, private translate: TranslateService) { }
+    private settingsService: ArlasSettingsService, private translate: TranslateService,
+    private arlasCollectionService: ArlasCollectionService) { }
 
   public export(contributor: Contributor, stayAtFirstLevel: boolean, contributorType?: string): Observable<Blob> {
     return this.compute(contributor, contributorType).pipe(map(data => {
       const csvData = new Array<Array<string>>();
       const header = new Array<string>();
       (<TreeContributor>contributor).getAggregations().forEach(agg => {
-        header.push(this.translate.instant(agg.field));
+        header.push(this.translate.instant(this.arlasCollectionService.getDisplayFieldName(agg.field)));
       });
       header.push(this.translate.instant('count'));
       if ((<TreeContributor>contributor).getAggregations()[0].metrics) {
         (<TreeContributor>contributor).getAggregations()[0].metrics.forEach(m => {
-          header.push(this.translate.instant('metric.'.concat(m.collect_field).concat('.').concat(m.collect_fct.toString())));
+          header.push(this.translate.instant(
```

Comment:
I think it is interesting to translate the header of the columns for the excel, but we will never know what needs translating and it will be a hassle translating this key. I suggest you use the variables of i18n, you can add a dictionary to the translate.instant method with all the other parameters